app/views/foreman_ansible/config_reports/_ansible.html.erb

```diff
@@ -29,8 +29,12 @@
                 <% end %>
               </ul>
             <% else %>
-              <%= log_message %>
-          <% end %>
+              <% allowed_length = 150 %>
+              <div class='pull-left'>
+                <%= truncate(log_message, :length => allowed_length) %>
+              </div>
+              <%= show_full_error_message_value(log_message) if log_message.length > allowed_length %>
```

Comment:
I think what you were looking for is the [trunc_with_tooltip](https://github.com/theforeman/foreman/blob/3347fa49d500964f0209122d8d36c920d1feafcc/app/helpers/layout_helper.rb#L175) helper.

I am not saying it's the best UX, but it's been used all across foreman and plugins. In addition, if we want to cherry-pick this to older versions, I would avoid adding the dependency PR in foreman 3.6, which will make the cherry-pick more difficult.

Reply:
The problem with `trunc_with_tooltip` helper, is that it doesn't display long texts properly. 
And as you can see in the attached [video](https://github.com/theforeman/foreman_ansible/pull/577#issue-1468313812), sometimes we are getting long error messages.  
So I can use that, but not all messages will display properly.

Reply:
yeah, your solution looks way better than the tooltip,
tested with Foreman and it works great!